rpcs3/Gui/GameViewer.h

```diff
@@ -88,12 +88,12 @@ struct ColumnsArr
 		m_img_list = new wxImageList(80, 44);
 
 		m_columns.clear();
-		m_columns.emplace_back(m_columns.size(),  90, "Icon");
+		m_columns.emplace_back(m_columns.size(),  90, "");
 		m_columns.emplace_back(m_columns.size(), 160, "Name");
 		m_columns.emplace_back(m_columns.size(),  85, "Serial");
-		m_columns.emplace_back(m_columns.size(),  55, "FW");
-		m_columns.emplace_back(m_columns.size(),  55, "App version");
-		m_columns.emplace_back(m_columns.size(),  75, "Category");
+		m_columns.emplace_back(m_columns.size(),  55, "Min fw");
```

Comment:
@paulsapps Which other? This is an old diff you're commenting on. And the reason for that would most likely be, that the row would take up most of the space in the game viewer.